printer_mods/README.md

```diff
@@ -138,5 +138,6 @@ like so:
 | | [V0 Electronics Cover](./xbst_/V0_Electronic_Panel)| A printed cover for v0 electronics. Requires larger printer to print. | :heavy_check_mark: | :x: | :x: | :x: |
 | | [V0 PiCam Mount](./xbst_/V0_Picam)| A Raspberry Pi Camera Mount For V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | | [DuetZero](./xbst_/DuetZero) | Pi Zero mount over a Duet2 for space saving | :heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: |:x:|
+| BladeScraper-Designs | [Microswitch-Pod-JST](./BladeScraper-Designs/Microswitch_Pod_JST) | Microswitch pod with JST-XH 4-pin connector | :x: |:x: |:heavy_check_mark: | :x:
```

Comment:
Please move this line up so that it's in alphanumeric order by username, not at the end of the file.

Reply:
I could have sworn I did but must have been on a different branch.  Fixed.